pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -446,13 +515,37 @@ func (r *reconciler) computeOFRulesForAdd(rule *CompletedRule, ofPriority *uint1
 	r.lastRealizeds.Store(rule.ID, lastRealized)
 
 	ofRuleByServicesMap := map[servicesKey]*types.PolicyRule{}
-
+	groupAddresses := r.getIGMPGroupAddress(rule)
+	isIGMP := len(groupAddresses) > 0
+	var from1, from2 []types.Address
+	if isIGMP && rule.Direction == v1beta2.DirectionIn {
+		// IGMP query
+		svcKey := servicesKey(fmt.Sprintf("igmp-query-%s", rule.ID))
+		ofPorts := r.getOFPorts(rule.TargetMembers)
+		lastRealized.podOFPorts[svcKey] = ofPorts
+		ofRuleByServicesMap[svcKey] = &types.PolicyRule{
+			Direction:     v1beta2.DirectionIn,
+			From:          append(from1, from2...),
```

Comment:
I don't get why we move from1 and from2 to L520 but never calculate it for IGMP but still fill it here.

Reply:
removed L520